README.md

```diff
@@ -1,55 +1,4 @@
 openmrs-distro-referenceapplication
 ===================================
 
-[master](https://github.com/openmrs/openmrs-distro-referenceapplication/) (platform-2.0.x): [![Build Status](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication/branches)
-
-[platform-2.1.x](https://github.com/openmrs/openmrs-distro-referenceapplication/tree/platform-2.1.x): [![Build Status](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication.svg?branch=platform-2.1.x)](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication/branches)
-
-## Running with OpenMRS SDK
-
-Please follow the instructions at the [OpenMRS SDK Wiki page](https://wiki.openmrs.org/display/docs/OpenMRS+SDK). Set up a server with the Reference Application distribution.
```

Comment:
Are we also getting away of using openMRS sdk. i think we still use this.

Reply:
The sub section of running with OpenMRS sdk was referring to running ui-test locally using a server set up via sdk. Now that these tests are no-longer in the module I think we can get rid of the section as well.